### PR TITLE
Sensor Relay Fixes

### PR DESCRIPTION
**Pull request by lavillastrangiato**
* Adds air alarms to every room in the sensor relay, thus ensuring that in case of a breach, the vents will function.
* Fixes the relay entrance airlock, adding two air canisters for it to fill and empty from.
* Fixes the relay air supply so it doesn't draw from a tank of pure oxygen.
* Makes the bathroom private, removing windows and glass airlocks from the bathroom.
  * Shuffles some windows and posters around.

### CODE REVIEW

**Comment by lavillastrangiato** [comment]
!review

**Review by DreamySkrell — APPROVED** [review]
image

lmao :skull: 

Mapping looks fine I think, approved.

**Review by alsoandanswer — APPROVED** [review]

File: html/changelogs/lavillastrangiato-PR-17313.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "Adds air alarms to every room in the sensor relay, thus ensuring that in case of a breach, the vents will function."
+  - maptweak: "Fixes the relay entrance airlock, adding two air canisters for it to fill and empty from."
+  - maptweak: "Fixes the relay air supply so it doesn't draw from a tank of pure oxygen."
+  - maptweak: "Makes the bathroom private, removing windows and glass airlocks from the bathroom."
```
Comment (SleepyGemmy) [inline html/changelogs/lavillastrangiato-PR-17313.yml:44]:
```suggestion
  - maptweak: "Makes the relay bathroom private, removing windows and glass airlocks from the bathroom."
```

File: html/changelogs/lavillastrangiato-PR-17313.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - maptweak: "Shuffles some windows and posters around."
```
Comment (SleepyGemmy) [inline html/changelogs/lavillastrangiato-PR-17313.yml:45]:
```suggestion
  - maptweak: "Shuffles some windows and posters around in the relay."
```

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**